include/uv/unix.h

```diff
@@ -38,7 +38,7 @@
 
 #if !defined(__MVS__)
 #include <semaphore.h>
-#include <sys/param.h> /* MAXHOSTNAMELEN on Linux and the BSDs */
+#include <sys/param.h> /* MAXHOSTNAMELEN on Linux and the BSDs; BSD on BSDs and Darwin */
```

Comment:
Final nit: please keep it <= 80 columns. ^^

Reply:
Sure thing; wasn't sure how to format it so I took a guess (how I've done it elsewhere). CONTRIBUTING.md's style guide section didn't specify how it should look for this particular case.

Feel free to nit further if there's a better way to achieve this. :)

Reply:
Looks fine to me 👍